src/lib.rs

```diff
@@ -743,6 +743,11 @@ impl RelativePathBuf {
     pub fn as_relative_path(&self) -> &RelativePath {
         self
     }
+
+    /// Consumes the `RelativePathBuf`, yielding its internal [`String`] storage.
```

Comment:
An example would be nice, but since it took so long for me to get to this it looks fine for now. Cheers!